pkg/volume/flexvolume/mounter.go

```diff
@@ -47,8 +47,8 @@ func (f *flexVolumeMounter) SetUp(fsGroup *int64) error {
 // SetUpAt creates new directory.
 func (f *flexVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 	call := NewDriverCall(f.plugin.getExecutable(), mountCmd, exec.New())
-	call.AppendSpec(f.spec, fsGroup, f.plugin.host)
```

Comment:
We are missing src here.. plugin might not supprt GetDeviceMountPath, in which case we need to pass the src path to bind mount to.